cypress/integration/Tooltip.spec.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="Cypress" />
+
+/* eslint-disable no-undef */
```

Comment:
Should be able to remove this, I think we have a `"no-undef": ["off"]` rule set in the cypress .eslintrc